modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -24,10 +24,14 @@ When you hover over the `i` to the right of each rule, the rule ID range for tha
 +
 image::waf-rule-set.png[]
 +
-. In *Advanced performance options*, you can select *Disable body scanner*. If you do not check this option, the request body is scanned.
+. In *Advanced performance options*, select from the following options: +
+* *Disable body scanner*. If you do not check this option, the request body is scanned.
 +
 [NOTE]
-The *Disable body scanner* option must be checked if any valid responses will exceed 512 KB. If request body checking is enabled, the HTTP Limits policy “Maximum Message Size” should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting the resources.
+The *Disable body scanner* option must be selected if any valid responses will exceed 512 KB. If request body checking is enabled, the HTTP Limits policy *Maximum Message Size* should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting resources.
```

Comment:
The first sentence is not accurate. First, I think "valid responses" should be "valid requests" since we are in the request section. Second, why must body scanning be disabled if requests exceed 512 KB? If a request body is larger than 1 MB, it will just not get scanned. No adverse side effects.